keras2onnx/main.py

```diff
@@ -131,6 +131,24 @@ def export_tf_frozen_graph(model, keep_var_names=None, output_names=None):
         return frozen_graph_def
 
 
+def _collect_input_nodes(graph, outputs):
+    nodes_to_keep = set()
+    input_nodes = set()
+    node_inputs = [graph.get_tensor_by_name(ts_).op for ts_ in outputs]
+    while node_inputs:
+        nd_ = node_inputs[0]
+        del node_inputs[0]
+        if nd_.type in ['Placeholder', 'PlaceholderWithDefault']:
```

Comment:
Are these two types the only types for tf inputs?

Reply:
extracted as a common func now.